<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-color: #fafafa; }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>961</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget { background: #fafafa; border: 2px solid #222;}</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="main_menu_page">
     <property name="styleSheet">
      <string notr="true">QWidget { background: #fafafa; }</string>
     </property>
     <widget class="QLabel" name="headline">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BlockEditor 1.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_scheme_btn">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>New Scheme</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_scheme_btn">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>260</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Scheme</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>some small description</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="help_page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>231</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Help page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="editor_page">
     <property name="styleSheet">
      <string notr="true">background: #fafafa;</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="menu">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Add Block</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Delete Block</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Add port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>STEP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>961</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="load_scheme_page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>170</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load *.scheme file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_file_btn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Choose a file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="open_file_btn">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>240</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="main_menu_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Main menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editor_menu_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Editor</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBlockEditor">
    <property name="title">
     <string>BlockEditor</string>
    </property>
    <addaction name="actionNew_Scheme"/>
    <addaction name="actionLoad_Scheme"/>
   </widget>
   <addaction name="menuBlockEditor"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Scheme">
   <property name="text">
    <string>New Scheme</string>
   </property>
  </action>
  <action name="actionLoad_Scheme">
   <property name="text">
    <string>Load Scheme</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
